api/server/sdk/volume_migrate.go

```diff
@@ -281,9 +284,11 @@ func (s *VolumeServer) Status(
 	}
 
 	// Filter out volumes we don't have access to
-	resp, err = s.filterStatusResponseForPermissions(ctx, resp)
-	if err != nil {
-		return nil, err
+	if auth.Enabled() {
```

Comment:
Since this call is expensive, we don't need to Enumerate all volumes only to permit all of them when auth is disabled.